sdk/signalr/azure-messaging-webpubsubservice/azure/messaging/webpubsubservice/__init__.py

```diff
@@ -165,24 +174,45 @@ def _format_url(self, url):
         assert self.endpoint[-1] != "/", "My endpoint should not have a trailing slash"
         return "/".join([self.endpoint, url.lstrip("/")])
 
-    def send_request(self, request, **kwargs):
+    def send_request(self, http_request, **kwargs):
         # type: (corerest.HttpRequest, Any) -> corerest.HttpResponse
         """Runs the network request through the client's chained policies.
 
-        :param request: The network request you want to make. Required.
-        :type request: ~corerest.HttpRequest
-        :keyword bool stream: Whether the response payload will be streamed. Defaults to False
-        :return: The response of your network call.
-        :rtype: ~corerest.HttpResponse
+        We have helper methods to create requests specific to this service in `azure.messaging.webpubsub.rest`.
+        Use these helper methods to create the request you pass to this method. See our example below:
+
+        >>> from azure.messaging.webpubsub.rest import build_healthapi_get_health_status_request
```

Comment:
I think you may want to wrap this in something like:
```
.. code-block:: python
```

Reply:
Good suggestion. @iscai-msft , we should make sure that the comments we generate render well in our docs. @annatisch , the format here is actually doctest compatible. Not sure if wrapping `.. code-block:: python` retains that.

Reply:
To me that would be weird for doctest to not test sections marked as code... but I could see nothing in the doctest docs that would indicate there might be issues.

Reply:
sounds good, making a note about this